eng/scripts/vcpkg.ps1

```diff
@@ -0,0 +1,52 @@
+[CmdletBinding()]
+Param (
+    [Parameter()]
+    [ValidateNotNullOrEmpty()]
+    [string] $Ref = 'master',
+
+    [Parameter(Mandatory = $true)]
+    [ValidateNotNullOrEmpty()]
+    [string] $Dependencies,
+
+    [Parameter()]
+    [string] $TargetPath ="$env:TEMP/$([guid]::NewGuid())",
```

Comment:
What directory is this? Shouldn't it be something like `$(Agent.TempDirectory)`?

Reply:
This was an old attempt at making this script potentially useful to someone working on their desktop. However stuffing vcpkg in a random place every time you run the script is a recipe for wasted time. 

It's been updated to check out in the root of the repo by default.